Dockerfile-datamgr

```diff
@@ -15,7 +15,7 @@
 FROM busybox:1.33.1 AS busybox
 
 FROM photon:4.0
-ADD /bin/linux/amd64/data-* /datamgr
+ADD /bin/linux/amd64/data-* /datamgr/
```

Comment:
Is this change needed?

Reply:
Without this change build will failed with error "When using ADD with more than one source file, the destination must be a directory and end with a /".